packages/osd-stylelint-config/config/restricted_properties.json

```diff
@@ -1,5 +1,6 @@
 {
   "font-family": {
+    "explanation": "OUI sets font families.",
```

Comment:
I'm a bit unsure what this should be or if it should even have a value. @joshuarrrr suggestions?